common/config_test.go

```diff
@@ -0,0 +1,92 @@
+package common
```

Comment:
I think, there are still few methos like AddPath and Reload, that needs Unit tests.

Perhaps best to add a //TO DO and an associated github issue to track the work to enhance the coverage of this important module!

Reply:
This is a more general issue -- our overall coverage is quite low at the moment. Definitely a good idea to have an issue for this, but probably doesn't make sense to have a TODO in this file specifically.